server/chat-commands/info.ts

```diff
@@ -2515,7 +2515,10 @@ export const commands: ChatCommands = {
 		if (!room.settings.requestShowEnabled) {
 			return this.errorReply(`Media approvals are disabled in this room.`);
 		}
-		if (user.can('showmedia', null, room)) return this.errorReply(`Use !show instead.`);
+		const showPerm = room.settings.permissions?.['/show'];
+		if (user.can('showmedia', null, room) && showPerm && room.auth.atLeast(user, showPerm)) {
+			return this.errorReply(`Use !show instead.`);
+		}
```

Comment:
```suggestion
		if (user.can('showmedia', null, room, '/show')) return this.errorReply(`Use !show instead.`);
```

This is easier, right?